website/static/js/osfLanguage.js

```diff
@@ -1,7 +1,7 @@
 var SUPPORT_EMAIL = 'support@osf.io';
-var SUPPORT_LINK = '<a href="mailto: ' + SUPPORT_EMAIL + '">' + SUPPORT_EMAIL + '</a>';
+var SUPPORT_EMAIL_MAILTO = '<a href="mailto:' + SUPPORT_EMAIL + '">' + SUPPORT_EMAIL +'</a>';
```

Comment:
It's possible that in the future we'll drive users to a form instead of an email link- can we change back to the old nomenclature?

(There are other PRs in progress as well that edit language.py, for which changing this name could lead to debugging headaches down the road)